frontend/tsconfig.json

```diff
@@ -6,8 +6,9 @@
       "dom.iterable",
       "esnext"
     ],
-    "types": ["node"],
-    "typeRoots": ["./src/@types"],
+    "baseUrl": ".",
+    "types": ["node", "jest", "@testing-library/jest-dom"],
+    // "typeRoots": ["./node_modules/@types"],
```

Comment:
jak macie w include **/* to tą linijkę można wywalić ;) Nie przyda się już raczej